File: pigatize.rb
def pigatize(text)
  if starts_with_vowel?(text)
    puts text + "way"
  else
    puts text +"ay"
  end
end

def starts_with_vowel?(text)
  vowels = %w[a e i o u]
  if vowels.include?(text[0].downcase)
    true
  else
    false
  end
end


puts pigatize("alaska")

File: pigatize.js

var startsWithAVowel = function (text) {
    var vowels = ["a", "e", "i", "o", "u"];
    for(var i= 0, l = vowels.length; i< l; i++){
        if(vowels[i] === text[0].toLowerCase() ){
  		      return true;
  	    }
    }
    return false;
};


var pigatize = function (text) {
  if (startsWithAVowel(text) === true) {
    return text + "way";
  } else {
    return text + "ay";
  }
};

console.log(pigatize("California"));
